### PR TITLE
Rewrote JSON API to use plain-old DTOs

### LINKED ISSUE #391: API: custom serializer `MemberJsonSerializer` is not used anymore
http://www.mix-it.fr/api/members/staff and http://www.mix-it.fr/api/members/staff?details=true produce same JSON result, from default Member entity serialization, not the one produced by `MemberJsonSerializer`.

### PR DESCRIPTION
**Pull request by clacote**
That is old-school and rebarbative work, but "it just works".

cc @javamind.
Fix #391 #351.

### CODE REVIEW

File: app/controllers/api/ApiMembers.java [hunk 1]
```diff
@@ -1,45 +1,113 @@
...
-    private static JsonSerializer SPONSOR_SERIALIZER = new SponsorJsonSerializer(false);
-    private static JsonSerializer STAFF_SERIALIZER = new StaffJsonSerializer(false);
-    private static JsonSerializer MEMBER_SERIALIZER = new MemberJsonSerializer(false);
-    private static JsonSerializer TALK_SERIALIZER = new TalkJsonSerializer(true);
+    private static final MemberToJsonVisitor SIMPLE_MEMBER_VISITOR = new AbstractMemberToDTOVisitor() {
+
+        @Override
+        public AbstractMemberDTO visit(Member member) {
+            MemberSimpleDTO dto = new MemberSimpleDTO();
+            initCommon(member, dto);
```
Comment (javamind) [inline app/controllers/api/ApiMembers.java]: This initCommon should be used in the methods initSimpleXXX

**Comment by javamind** [comment]
:+1:

File: app/controllers/api/ApiMembers.java (hunk 1 above)
Comment (clacote) [inline app/controllers/api/ApiMembers.java]: Fixed in f5785ac

**Comment by clacote** [comment]
Also, I fixed DateTime marshalling (previously used in `AbstractSessionDTO`): replaced `DateTime` types to `String`, to avoid obscure default `DateTime` marshalling.

**Comment by clacote** [comment]
@javamind: any chance that you could test your mobile app with a local deployment to detect any regression?

**Comment by javamind** [comment]
@clacote difficult before tonight because I have a new laptop and I have'nt Play Framework for the moment

**Comment by javamind** [comment]
:clap: :clap: 
ok it works

**Comment by clacote** [comment]
So merge'n'deploy? I'm legally not allowed to merge my own pull requests. But I can do it :p

**Comment by clacote** [comment]
There is nobody now on mix-it.fr: i'm merging and deploying.

**Pull request merged**